Source/WebKit/UIProcess/Gamepad/gtk/UIGamepadProviderGtk.cpp

```diff
@@ -68,22 +68,34 @@ static WebPageProxy* getWebPageProxy(GtkWidget* widget)
     return nullptr;
 }
 
+static bool windowHasTopLevelFocus(GtkWindow* window)
+{
+#if ENABLE(DEVELOPER_MODE)
+    // Xvfb doesn't support toplevel focus, so we consider toplevel window to be always active since it's the only one.
+    if (Display::singleton().isX11()) {
+        static const char* underXvfb = g_getenv("UNDER_XVFB");
+        if (!g_strcmp0(underXvfb, "yes"))
+            return true;
+    }
+#endif
```

Comment:
Can't this go to a single place in some way? I see this comment in at least 4 or 5 places with the same checks.

Reply:
Yes, but in a follow up patch.

Reply:
Yes, please 🙏🏼